PythonToolbox/quantconnect/order.py

```diff
@@ -0,0 +1,18 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ORDER_TYPES = ['Market', 'Limit', 'StopMarket', 'StopLimit', 'MarketOnOpen', 'MarketOnClose', 'OptionExercise']
+
+ORDER_DIRECTIONS = ['Buy', 'Sell', 'Hold']
+
+ORDER_STATUSES = ['New', 'Submitted', 'PartiallyFilled', 'Filled', None, 'Canceled', 'None', 'Invalid', 'CancelPending', 'UpdateSubmitted']
```

Comment:
nit - we might add a comment of why there is an extra `None` in there (spoiler: because the C# enum jumps from 3 to 5) through we off there for a second 😄